src/main/java/duke/TaskManager.java

```diff
@@ -0,0 +1,102 @@
+package duke;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+public class TaskManager {
+    private static final String TODO = "todo";
+    private static final String DEADLINE = "deadline";
+    private ArrayList<Task> list = new ArrayList<>(100);
```

Comment:
Maybe renaming the ArrayList as tasks would be more clear in showing the content of the list?